src/styles/CountryDetails.module.css

```diff
@@ -0,0 +1,42 @@
+.banner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+  overflow: hidden;
+}
+
+img {
+  width: 100%;
+  max-height: 50vh;
+}
```

Comment:
- [ ] good styling, but this could be made more reusable by putting it in a more global style file instead of repeating it  in countryList.module.css, if something is different, it can den be created in the modules :+1:

Reply:
Hi @VICTOR-CHUKWU, Noted with thanks!